### PR TITLE
Progress till 10-02-22

### PR DESCRIPTION
**Pull request by sadab-halim**

### CODE REVIEW

**Comment by kranurag7** [comment]
@sadab-halim I am pointing out some typos. Don't feel irritated. Just approve the changes. I think this will help you in future when you will be contributing to big projects.

File: Sadab-Halim/10-02-22/AWS_Cloud.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+**Storage included** <br/>
+1 GB of storage free per AWS region
+
+**Saved files and settings** <br/>
+Files saved in you home directory are available in future sessions for the same AWS region
+
+**Shell Environments** <br/>
+Seamlessly switch between
+- Bash
+- PowerShekk
```
Comment (kranurag7) [inline Sadab-Halim/10-02-22/AWS_Cloud.md:64]:
```suggestion
- Powershell
```

File: Sadab-Halim/10-02-22/AWS_Cloud.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- Zsh
+
+## Infrastructure as Code (IaC)
+You write a configuration script to automate creating, updating or destroying cloud infrastructure.
+- IaC is a blueprint of your infrastructure.
+- IaC allows you to easily sharem version or inventory your cloud infrastructure.
```
Comment (kranurag7) [inline Sadab-Halim/10-02-22/AWS_Cloud.md:70]:
```suggestion
- IaC allows you to easily share version or inventory your cloud infrastructure.
```

File: Sadab-Halim/10-02-22/AWS_Cloud.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+- Less verbosem you could end up with misconfiguration
+- Does more than Declarative
+- Uses programming languages eg. Python Rubym JavaScript
+
+## CloudFormation
+AWS CloudFormation allows you to write Infrastructure as Code (IaC) as either a JSON or YAML file.
+
+CloudFormation is simple but it can lead to large files or is limited in some regard to creating dynamic or repeatable infrastructure compared to CDK.
+
+CloudFormation can be easier for DevOps Engineers who do not have a backgroung in web programming languages.
```
Comment (kranurag7) [inline Sadab-Halim/10-02-22/AWS_Cloud.md:96]:
```suggestion
CloudFormation can be easier for DevOps Engineers who do not have a background in web programming languages.
```

File: Sadab-Halim/10-02-22/AWS_Cloud.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+- What you see is what you get. **_Explicit_**
+- More verbose, but zero chance of mis-configuration
+- Uses scripting languages eg. JSON, YAML, XML
+
+**AWS Cloud Development Kit (CDK)**
+CDK is an Imperative IaC tool.
+
+**Imperative**
+- You say what you want, and the rest is filled in. **_Implicit_**
+- Less verbosem you could end up with misconfiguration
```
Comment (kranurag7) [inline Sadab-Halim/10-02-22/AWS_Cloud.md:87]:
```suggestion
- Less verbose you could end up with misconfiguration
- ```

File: Sadab-Halim/10-02-22/AWS_Cloud.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+- Does more than Declarative
+- Uses programming languages eg. Python Rubym JavaScript
```
Comment (kranurag7) [inline Sadab-Halim/10-02-22/AWS_Cloud.md:89]:
```suggestion
- Uses programming languages eg. Python, Ruby & JavaScript.
```

**Review by kranurag7 — CHANGES_REQUESTED** [review]

**Review by Rajendraladkat1919 — COMMENTED** [review]
Looks good to me.

**Comment by kranurag7** [comment]
Thanks for review @Rajendraladkat1919

**Review by sadab-halim — COMMENTED** [review]
Thanks @kranurag7 for the typos.
I have made the changes.

**Pull request merged**